cp1404practicals/prac_05/emails.py

```diff
@@ -0,0 +1,21 @@
+"""
+Emails
+
+Estimate: 30 minutes
+Actual  : 30 minutes
+"""
+
+email_dictionary = {}
+email = input("Email: ")
+
+while email != "":
+    name = " ".join(email.split("@")[0].split(".")).title()
+    choice = input(f"Is your name {name}? (Y/n) ").lower()
+    if choice != "" and choice != "y":
+        name = input("Name: ").title()
+    email_dictionary[name] = email
+    email = input("Email: ")
+
+for email in email_dictionary.items():
+    print(email)
+    print(f"{email[0]} ({email[1]})")
```

Comment:
Hi bro, i do not know how to do this question tbh, but I used your code as reference. You method helped me a lot!

Reply:
No worries bro feel free